doc/modules/decomposition.rst

```diff
@@ -861,19 +861,46 @@ discovering abstract topics from a collection of documents.
 
 The graphical model of LDA is a three-level Bayesian model:
 
-.. image:: ../images/lda_model_graph.png
+.. image:: ../images/lda_model_graph2.png
    :align: center
 
+
+Note on notations presented in the graphical model above:
+
+  * The corpus is a collection of :math:`D` documents.
+  * A document is a sequence of :math:`N` words.
+  * There are :math:`K` topics in the corpus. 
+  * The boxes represent repeated sampling. 
+
+
+The three levels of the Bayesian model are: 
+
+  1. Generation of global hyperparameters :math:`\alpha` and :math:`\eta`, and global parameter :math:`\beta`.
+
+  2. Generation of document-level variable :math:`\theta`. 
```

Comment:
Can you make it explicit that theta corresponds to the `topic_word_prior` parameter?
Same for other non-obvious names

Reply:
Done. Please see commit f139d2d